packages/bezier-react/src/components/Center/Center.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+import {
+  type Meta,
+  type StoryFn,
+} from '@storybook/react'
+
+import { Center } from './Center'
+import { type CenterProps } from './Center.types'
+
+const meta: Meta<typeof Center> = {
+  component: Center,
+}
+
+const Template: StoryFn<CenterProps> = ({ children, ...rest }) => (
+  <Center {...rest}>
+    { children }
+  </Center>
+)
+
+export const Primary = {
```

Comment:
```suggestion
export const Primary: StoryObj<typeof meta> = {
```

타입이 누락되어 bgColor속성에 type error가 뜨지 않았습니다.